libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -84,55 +85,56 @@ - (id)init {
         if (self.rawPreferences) {
             [[SalesforceSDKManager sharedManager] registerAppFeature:kSFAppFeatureManagedByMDM];
         }
-        NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
-        if (self.rawPreferences) {
-            attributes[@"mdmIsActive"] = [NSNumber numberWithBool:YES];
-            attributes[@"mdmConfigs"] = self.rawPreferences;
-        } else {
-            attributes[@"mdmIsActive"] = [NSNumber numberWithBool:NO];
-        }
-        [SFSDKEventBuilderHelper createAndStoreEvent:@"mdmConfiguration" userAccount:nil className:NSStringFromClass([self class]) attributes:attributes];
+        NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
+        [notificationCenter addObserver:self
+                               selector:@selector(storeAnalyticsEvent)
+                                   name:SFUserAccountManagerDidFinishUserInitNotification
+                                 object:nil];
     }
     return self;
 }
 
-- (BOOL)hasManagedPreferences {
+- (void) dealloc {
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+- (BOOL) hasManagedPreferences {
     return ([self.rawPreferences allKeys].count > 0);
 }
 
-- (BOOL)requireCertificateAuthentication {
+- (BOOL) requireCertificateAuthentication {
     return [self.rawPreferences[kManagedKeyRequireCertAuth] boolValue];
 }
 
-- (BOOL)onlyShowAuthorizedHosts {
+- (BOOL) onlyShowAuthorizedHosts {
     return [self.rawPreferences[kManagedKeyOnlyShowAuthorizedHosts] boolValue];
 }
 
-- (NSArray *)loginHosts {
+- (NSArray *) loginHosts {
     id objLoginHosts = self.rawPreferences[kManagedKeyLoginHosts];
     if ([objLoginHosts isKindOfClass:[NSString class]]) {
         objLoginHosts = @[ objLoginHosts ];
     }
     return objLoginHosts;
 }
 
-- (NSArray *)loginHostLabels {
+- (NSArray *) loginHostLabels {
     id objLoginHostLabels = self.rawPreferences[kManagedKeyLoginHostLabels];
     if ([objLoginHostLabels isKindOfClass:[NSString class]]) {
         objLoginHostLabels = @[ objLoginHostLabels ];
     }
     return objLoginHostLabels;
 }
 
-- (NSString *)connectedAppId {
+- (NSString *) connectedAppId {
     return self.rawPreferences[kManagedKeyConnectedAppId];
 }
 
-- (NSString *)connectedAppCallbackUri {
+- (NSString *) connectedAppCallbackUri {
     return self.rawPreferences[kManagedKeyConnectedAppCallbackUri];
 }
 
-- (BOOL)shouldDisableExternalPasteDefinedByConnectedApp {
+- (BOOL) shouldDisableExternalPasteDefinedByConnectedApp {
```

Comment:
Is there a reason we're adding these spaces between the return type and method names?  The former is the general style convention for Objective-C coding.